packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -345,6 +345,7 @@ exports.createNodesForContentType = ({
 
         if (
           fieldProps.type === `RichText` &&
+          richTextOptions &&
```

Comment:
I could also get rid of the `.default({})` in [`plugin-options.js`](https://github.com/gatsbyjs/gatsby/blob/81bf2d74fd23911da724551d479d74192f44e1f1/packages/gatsby-source-contentful/src/plugin-options.js#L55) since it doesn't seem to be having any effect. Thoughts?

Reply:
the `defaults` there just seem to ensure that `richText` object exists and we don't have to make sanity checks before accessing?

Reply:
oh well, nvm I just read code around here

Reply:
let's merge this in as hot fix and figure out the .defaults situation later

Reply:
so what I think happens is that we only is when we call `validate` in https://github.com/gatsbyjs/gatsby/blob/81bf2d74fd23911da724551d479d74192f44e1f1/packages/gatsby-source-contentful/src/plugin-options.js#L61

it doesn't do any changes to `options` that is argument to that

`validate` return object with `error` and `value` (value will have normalized things), but we never use `value` in `pluginOptions` (just check the `error`)